Automattic/jetpack
# Beta: Soft fail if Composer pkgs not installed

#### Changes proposed in this Pull Request:

* Provide a soft fail if activating Beta without having `jetpack install plugins/beta`.

This is the same experience as Jetpack in the same situation.

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* `jetpack clean plugins/beta` and clean everything.
* In Docker, activate Beta. See hard fatal.
* On this branch, same thing. See error message.
* `jetpack install plugins/beta`
* See the plugin work.
